components/ChallengeMaterial.tsx

```diff
@@ -200,12 +221,11 @@ const ChallengeMaterial: React.FC<ChallengeMaterialProps> = ({
     status: 'passed'
   }
   //find first challenge that is not passed on initial render after clicks will render clicked challenge
-  const currentChallenge: ChallengeSubmissionData =
+  let currentChallenge =
```

Comment:
Why let?

Reply:
good catch! this must've got changed during the rebase and  i didn't catch it :)

Reply:
fixed :)